README.md

```diff
@@ -201,6 +201,25 @@ In the VS Code extension (version 1.3.7+) set the `toitLanguageServer.command` s
 
 This makes the extension use the language server that was compiled in the [build step](#build-for-host-machine).
 
+### Jetbrains IntelliJ Platform
```

Comment:
Suggestion: Jetbrains' IntelliJ platform

The other titles in this README do not use title case, so for consistency I don't think this one should either.